src/value.cpp

```diff
@@ -1608,6 +1608,11 @@ MemRef MemRef::subview(const vector<Expr> &offsets,
   }
 }
 
+MemRef MemRef::reshape(const std::vector<smt::Expr> &newDims) {
```

Comment:
Should we insert an assertion stating that this MemRef has an identity layout?

Reply:
resolved